modyn/tests/selector/internal/selector_strategies/downsampling_strategies/test_gradnorm_downsampling_strategy.py

```diff
@@ -0,0 +1,36 @@
+from modyn.selector.internal.selector_strategies.downsampling_strategies import GradNormDownsamplingStrategy
+
+
+def test_init_gradnorm():
+    # Test init works
+    strat = GradNormDownsamplingStrategy(
+        {
+            "ratio": 10,
+            "sample_then_batch": True,
+        },
+        1000,
+    )
+
+    assert strat.downsampling_ratio == 10
+    assert isinstance(strat.remote_downsampling_strategy_name, str)
+    assert strat.requires_remote_computation
+
+
+def test_command_gradnorm():
```

Comment:
sorry, what does that test do in comparison to the init test? it initializes it the same way, right?

Reply:
The second test made sense the way the classes were structured before. Now I can combine them

Reply:
Ok top :)